src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/core/AbstractCraftingProductionResolver.java

```diff
@@ -148,13 +148,13 @@ protected boolean canBuildingCraftStack(@NotNull final IRequestManager manager,
                 {
                     materialRequests.add(createNewRequestForStack(manager, craftingHelperStack, ingredient.getAmount(), ingredient.getAmount()));
                 }
-                else if (!ItemStackUtils.isEmpty(container) && ItemStackUtils.compareItemStacksIgnoreStackSize(container, craftingHelperStack, false, true))
+                else if (ItemStackUtils.isEmpty(container) || ItemStackUtils.compareItemStacksIgnoreStackSize(container, craftingHelperStack, false, true))
```

Comment:
Why this change? Documentation is missing here. The original code makes more sense....

Reply:
reverted